api/v2alpha1/condition_types.go

```diff
@@ -51,20 +51,14 @@ const (
 	// ReadyCondition represents the fact that the HelmRelease has been successfully reconciled.
 	ReadyCondition string = "Ready"
 
-	// InstalledCondition represents the fact that the HelmRelease has been successfully installed.
-	InstalledCondition string = "Installed"
-
-	// UpgradedCondition represents the fact that the HelmRelease has been successfully upgraded.
-	UpgradedCondition string = "Upgraded"
+	// ReleasedCondition represents the fact that the HelmRelease has been successfully released.
+	ReleasedCondition string = "Released"
 
 	// TestedCondition represents the fact that the HelmRelease has been successfully tested.
 	TestedCondition string = "Tested"
```

Comment:
Wondering if something like

```suggestion
	TestedCondition string = "Healthy"
```

would provide a better context to what the condition represents, as `Tested==False` means something entirely different to me on a first glance than what it actually represents.

Reply:
`TestSucceeded`?

Reply:
Depends on what it will be used for, if we implement test runs on an interval in a next iteration, the `Healthy` will make more sense I think.

Reply:
I think the only real use case for this condition is when tests are ignored, since otherwise the `Ready` condition should suffice as it would contain `TestFailed` as the reason. In this case the release may be healthy but the tests are just flaky. I could also imagine a future feature where either helm itself (e.g. a new `helm status` or `helm wait` command) or the helm-controller implement continuous status checking of the resources in a release. In which case we'd want to distinguish this condition from that feature. So we may want to make it clear that this is specific to helm tests.  

I do like the idea of using tense for the interval based test support. Some options:

1. `TestsHealthy`
2. `TestsPassing`
3. `TestsSucceeding`

Reply:
I think 2 is a nice balance between "human readable" and "machine readable".

Reply:
Went with `TestSuccess` for now to be consistent with the `TestSucceeded` condition reason, which is consistent with e.g. `InstallSucceeded`, `UpgradeSucceeded`, , `RollbackSucceeded`, `UninstallSucceeded`.